packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -127,7 +164,11 @@ export function BlockTypesTab(
 					return (
 						<InserterPanel
 							key={ category.slug }
-							title={ category.title }
+							title={
+								categoryItems.length === 1
+									? getRootBlockTitle()
+									: category.title
```

Comment:
I don't understand this condition. Why do we fall back to the title of the parent of the selected block when there's 1 category?

Reply:
Ah, I got confused by `categoryItems`, I'm guessing this is just the total amount of items, not the amount of categories.